### ISSUE TITLE
Proptypes warning when passing single style from stylesheet

### ISSUE THREAD

**Issue by adamterlson** [react-native-fellowship/react-native-navbar#98]
The propTypes for NavigationBar is not expecting a singular stylesheet reference.

````
let styles = StyleSheet.create({ ... });
...
      <NavigationBar
        style={styles.myStyle} />
````

This is likely because `styles.myStyle` is in fact a number, not an object.

To work around this issue, you can define style as `[{styles.myStyle}]`.

**Comment by Kureev** [comment]
Don't see any question here

**Comment by adamterlson** [comment]
@Kureev Sorry, does that mean that the issue is not clear?  The PropTypes needs to include the case where the style being passed in is a number to support stylesheets being used.

**Comment by Kureev** [comment]
Well, I've never used StyleSheets before, so I can't get why you complain about it :)

**Comment by Kureev** [comment]
Fixed in 1.0.7

**Comment by adamterlson** [comment]
(+1)
